### ISSUE TITLE
Script error during processing destruction with ?? operator

### ISSUE THREAD

**Issue by miherlosev** [DevExpress/testcafe-hammerhead#2782]
### Origin code
```js
function aria() {
            var {icon: t, text: n} = this.props
                , r = (n ?? "") || t;
        }
```

### Processed code
```js
 function aria() {
             var _hh$temp0=this.props,t=_hh$temp0.icon,n=_hh$temp0.text,r=n??""||t; 
 }
```

**Comment by jlabaj** [comment]
hi @miherlosev  will this fix(testcafe-hammerhead v24.5.24) be a part of any alfa of testcafe soon?

**Comment by Aleksey28** [comment]
Hi @jlabaj,

It is already released in the 1.20.0-rc.1 Testcafe version.
